.github/workflows/python-build-test.yaml

```diff
@@ -202,6 +205,7 @@ jobs:
       - name: Install pylace
         if: ${{ matrix.target != 'aarch64' }}
         run: |
+          ls -l ./dist
```

Comment:
Debugging code left in?

Reply:
Yeah, I was trying to figure out what was going on. Just fyi, `macos-14` runner is only `aarch64`.